### PR TITLE
Services/diseases ner/migrate

### PR DESCRIPTION
**Pull request by leogail**

### CODE REVIEW

File: services/diseases-ner/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# ws-diseases-ner@1.0.7
+
+Trouve des entités nommées de maladies dans un texte
+
+Renvoie un Json composé comportant un champs `diseases` correspondant aux entités nommées de maladies trouvées
```
Comment (parmentf) [inline services/diseases-ner/README.md:5]: **typo:** ‏
```suggestion
Renvoie un Json composé comportant un champ `diseases` correspondant aux entités nommées de maladies trouvées
```

File: services/diseases-ner/config.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "environnement": {
+        "EZS_TITLE": "Trouve des entités nommées de maladies dans un texte",
+        "EZS_DESCRIPTION": "Renvoie un Json composé comportant un champs `diseases` correspondant aux entités nommées de maladies trouvées",
```
Comment (parmentf) [inline services/diseases-ner/config.json:4]:
```suggestion
        "EZS_DESCRIPTION": "Renvoie un Json composé comportant un champ `diseases` correspondant aux entités nommées de maladies trouvées",
```

File: services/diseases-ner/examples.http [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# These examples can be used directly in VSCode, using REST Client extension (humao.rest-client)
+# They are important, because used to generate the tests.hurl file.
+
+# Décommenter/commenter les lignes voulues pour tester localement
+@host=http://localhost:31976
+# @host=https://diseases-ner.services.istex.fr
+
+###
+# @name v1routeInCamelCase
+# Description de la route
```
Comment (parmentf) [inline services/diseases-ner/examples.http:10]: **nitpick (non-blocking):** ‏Adapt name and description to the service

```suggestion
# @name v1diseasesTagger
# Trouve les entités nommées de maladies
```

File: services/diseases-ner/tests.hurl [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+		"value": "They have been used in the prevention and treatment of malaria and autoimmune diseases, including systemic lupus erythematosus and rheumatoid arthritis." 
+	}
+]
+
+
+HTTP 200
+[{
+    "id": 1,
+    "value": {
+        "diseases": [
```
Comment (parmentf) [inline services/diseases-ner/tests.hurl:15]: **thought (non-blocking):** Pourquoi imbriquer `diseases` dans `values`

You return only one field, why not use `value`, and put the array at this level?
Do you plan to add other fields?
See [YAGNI](https://fr.wikipedia.org/wiki/YAGNI)

File: services/diseases-ner/v1/diseases/tagger.ini [hunk 5]
```diff
@@ -0,0 +1,42 @@
+# OpenAPI Documentation - JSON format (dot notation)
+mimeType = application/json
+
+post.description = Trouve des entités nommées de maladies dans un texte
+post.responses.default.description = Renvoie un Json composé comportant un champs `value` correspondant aux entités de maladies trouvées
```
Comment (parmentf) [inline services/diseases-ner/v1/diseases/tagger.ini:5]:
```suggestion
post.responses.default.description = Renvoie un Json composé comportant un champ `value` correspondant aux entités de maladies trouvées
```

**Review by parmentf — CHANGES_REQUESTED** [review]

File: services/diseases-ner/tests.hurl (hunk 4 above)
Comment (leogail) [reply services/diseases-ner/tests.hurl:15]: Two reasons : 
- being the most consistent with other ws doing NER
- Can add a field "disease_disambiguisate" later without breaking change

**Review by parmentf — APPROVED** [review]

**Pull request merged**